tsb/cp/providers.tf

```diff
@@ -8,22 +8,22 @@ terraform {
 }
 
 provider "kubernetes" {
-  host                   = local.infra[var.tsb_mp["cloud"]][var.tsb_mp["cluster_id"]]["outputs"].host
-  cluster_ca_certificate = base64decode(local.infra[var.tsb_mp["cloud"]][var.tsb_mp["cluster_id"]]["outputs"].cluster_ca_certificate)
-  token                  = local.infra[var.tsb_mp["cloud"]][var.tsb_mp["cluster_id"]]["outputs"].token
+  host                   = local.infra["outputs"].host
+  cluster_ca_certificate = base64decode(local.infra["outputs"].cluster_ca_certificate)
+  token                  = local.infra["outputs"].token
```

Comment:
@smarunich PTAL, I think this setup was fundamentally wrong, as we're deploying CPs here but the variables referenced the cluster_id referenced from the `mp` settings... I may be missing something, but... how could this be working when more than 1 cluster was deployed? What am I missing?

In any case I think the change is correct and the current configuration is the right one.

Reply:
indeed, it could not be working in the past, in fact the tsb_cp module k8s provider being initialized as part of tsb_cp module itself, the outer `providers.tf` is used for k8s operations outside of the modules which we don't have any yet at this stage, so it is a more ready to go placeholder at this stage.